ee/codegen/src/__test__/__snapshots__/inputs.test.ts.snap

```diff
@@ -2,79 +2,93 @@
 
 exports[`Inputs > write > should convert input variable names into valid python attributes 1`] = `
 "from vellum.workflows.inputs import BaseInputs
-from typing import Union
+from typing import Optional, Union
 
 
 class Inputs(BaseInputs):
-    my_input: str
-    my_input_1: Union[float, int]
+    my_input: Optional[str]
+    my_input_1: Optional[Union[float, int]]
```

Comment:
[nit] I'm a bit worried about all of these unrelated test snapshots updating. were we not using `"required": true` in the other fixtures?
- If not, let's add to minimize unrelated tests updating
- If so, then this would hint at a bug in our impl to investigate